builtin/logical/transit/backend.go

```diff
@@ -47,6 +50,7 @@ func Backend(conf *logical.BackendConfig) *backend {
 			b.pathBackup(),
 			b.pathRestore(),
 			b.pathTrim(),
+			b.pathCacheConfig(),
```

Comment:
Is there a reason this should be part of a config path instead of a mount option?

Reply:
The new cache type is only available to the transit backend so I thought it would be confusing to have a mount option that only applies to one kind of mount (but it could be if that seems more reasonable).

Reply:
In theory this is exactly what the options map is for. I think the main issue is plumbing them into CLI, if we start adding options only valid for some backends it could make the CLI kind of ugly. This might be a good topic for the offsite as it's come up a few times in various ways, let's talk there!